Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/AppContentView.swift

```diff
@@ -25,7 +25,7 @@ struct AppContentView: View {
 
     var body: some View {
         TabView {
-            if Purchases.isConfigured {
+            if false && Purchases.isConfigured {
```

Comment:
I'd add a comment above this explaining the reasoning behind it being disabled, otherwise to a maintainer this looks like an accident

Reply:
Done!